AwaMelvine/fast-food-fast
# Ft items image upload 162180514

#### What does this PR do?
Implements image upload on food items.
#### Description of Task to be completed?
- Include the image field in the create food item request when submitting to the server
- Delete food item image when deleting a food item

#### How should this be manually tested?
- Having logged in as an admin user, visit the create food item page on the website
- Create a new food item by filling out the add food item form, select an image for the item
- Click on create a food item
- Check the existing items page in the public section and check the new post you created to make sure it displays with the image that was selected

#### What are the relevant pivotal tracker stories?
[#162180514] User should be able to upload an image from the client
